### PR TITLE
GitHub GUI tutorial

### PR DESCRIPTION
**Pull request by mankoff**
Image links don't work (yet) because they're to `main` not the branch (which will eventually be deleted). Not sure best-practices here.

I also want to set up some links between md files, but again, on main not a branch, so the links don't exist yet.

Just setting up this PR so you can have a glance at this draft, comment on level of detail, scope, style, etc.

### CODE REVIEW

File: doc/tutorial/working-with-GitHub/50-working-with-existing-repositories.md [hunk 1]
```diff
@@ -3,20 +3,32 @@
...
 
-### Editing the file with GitHub.com
+## Edit the file
 
 For simple edits, use edit with GitHub.com
 
-1. Click on the `pen` icon, or select `Edit this file` from the `v` dropdown next to the pen icon.
+1. Click on the `pen` icon, or select `Edit this file` from the `v` dropdown next to the pen icon
+
+![Edit a file](https://github.com/nsidc/git-training/raw/main/doc/_images/gh-edit-file.png)
```
Comment (MattF-NSIDC) [inline doc/tutorial/working-with-GitHub/50-working-with-existing-repositories.md]: I believe the standard way to reference an image that's part of the source is `Text`

Comment (mankoff) [reply doc/tutorial/working-with-GitHub/50-working-with-existing-repositories.md]: Or in this case, `../../`

Comment (MattF-NSIDC) [reply doc/tutorial/working-with-GitHub/50-working-with-existing-repositories.md]: Poked at it a bit more, and `/_images/the-image.png` works too. I pushed a couple new commits and the github integration is working now :man_dancing:

**Comment by MattF-NSIDC** [comment]
LGTM, thanks Ken!

**Pull request merged**